includes/helper-functions.php

```diff
@@ -26,6 +26,11 @@
  * @since 1.0
  * @return object Dismissible_Notices_Handler
  */
+
+if ( function_exists( 'DNH' ) ) {
```

Comment:
The idea is certainly valid but given that this is in the PHP file directly (and not inside a function) I think that declaring or not declaring the function makes more sense than trying to prevent the rest of the content from loading.

```suggestion
if ( ! function_exists( 'DNH' ) ) {
	function DNH() {
		return Dismissible_Notices_Handler::instance();
	}
}
```

Reply:
I think that is better to not load all the file at all as the error will happen also for them.

So a check for the first function and block the loading of the rest of the file was a most simple way to maintain everything.

Reply:
In a scenario where an older version of the library would be loaded without the functions following the `return` statement, should there be another plugin/library/theme relying on those functions but their (newer) version of DNH is not loaded (an older version was loaded first) then it would trigger an error for calls to non-existing functions.

Reply:
the code between versions is the same, as example in the discussion the fork mentioned just silenced some warnings.
So block the loading of all the functions also if it is an old or a not update fork there would not be errors, as it will be loaded another library that will works in the same way.

Reply:
I just want that this error doesn't happen again to my plugin as I have already 2 user reported the issue and seems that the other plugin doesn't care and the code of this is kind of stable as api and in the last years didn't changed.